### PR TITLE
(DOCSP-45011) Run a Command

### PR DESCRIPTION
**Pull request by elyse-mdb**
# Pull Request Info
Create a page that explains command options and how to run a command. Similar to: https://www.mongodb.com/docs/drivers/node/current/fundamentals/run-command/

[PR Reviewing Guidelines](https://github.com/mongodb/docs-java/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-45011>
Staging - <https://deploy-preview-70--docs-cpp.netlify.app/run-command/>

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/run-command.txt [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+   method inside the shell is the preferred method to issue database
+   commands, as it provides a consistent interface between the shell and
+   drivers.
+
+Execute a Command
+-----------------
+
+To run a database command, call the ``run_command()`` execution method on a ``mongocxx::database`` instance 
+and pass in a document that specifies the command and any relevant arguments. 
+The method will return the result of the command that was run as a ``bsoncxx::document::value`` object.
```
Comment (jordan-smith721) [inline source/run-command.txt:38]: Remove future-tense and passive voice
```suggestion
The method returns the result of the command as a ``bsoncxx::document::value`` object. 
```

File: source/run-command.txt [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+   * - ``ok``
+     - Indicates whether the command has succeeded (``1``)
+       or failed (``0``).
+
+   * - ``operationTime``
+     - Indicates the logical time of the operation. MongoDB uses the
+       logical time to order operations.
+       
+       To learn more about logical time, see our :website:`blog post about
+       the Global Logical Clock </blog/post/transactions-background-part-4-the-global-logical-clock>`.
```
Comment (jordan-smith721) [inline source/run-command.txt:170]: This link goes to the transactions page of the Server manual instead of a blog post. I would suggest rewording this if the transactions page is what you're trying to link to, or correct the link to point to the blog post

**Review by jordan-smith721 — APPROVED** [review]
Nice work! LGTM with two small fixes

File: source/run-command.txt (hunk 2 above)
Comment (elyse-mdb) [reply source/run-command.txt:170]: Ah, it looks like the original blog post is either no longer up or no longer accessible. (A post called **Transactions Background Part 4: The Global Logical Clock**). I removed this link from the page.

**Review by kevinAlbs — APPROVED** [review]
